style.css

```diff
@@ -303,7 +334,7 @@ label {
   background: #fff;
 }
 
-@media only screen and (min-width: 600px) {
+@media only screen and (min-width: 1280px) {
```

Comment:
- [Optional] Dear @Feliverse, I see you have changed your media query sizes. this is causing a problem. the `hamburger ` icon better be hidden for a screen wider than `768px`. your design seems to act as a mobile on a screen size of `1263px`. this would make your design poor. please make appropriate changes to make your design even better. 
- [ ] ![your-desktop](https://user-images.githubusercontent.com/69077061/201506681-37458894-1d8e-4894-8fb8-ff33790956f3.png)